woocommerce/woocommerce-android
# [TTP] Move set up tap to pay to a separate section and show request for feedback if its been used before

Closes: #8458


### Description

Design UsEjcaCj3qJ5pqnXeYz7dq-fi-329%3A19231

The PR:
* Adjusts the UI to match the design
* Adds request for feedback if TTP was used at least once
The request for feedback button does nothing so far

### Testing instructions

* Open the Payments hub and notice that request for TTP feedback is missing
* Run any TTP flow (testing/simple payment/order)
* Open the Payments hub and notice that request for TTP feedback is there

### Images/gif

<img src=https://user-images.githubusercontent.com/4923871/224919737-0775009c-df38-44a4-b65e-88c12fffd1a2.png width=300/>
<img src=https://user-images.githubusercontent.com/4923871/224919744-bed937fe-30b1-4e56-9745-d2325e7e6c0e.png width=300/>
